### PR TITLE
initiated a `docsify` website at `docs`

### PR DESCRIPTION
**Pull request by dev-matan-tsuberi**
Created a [docsify](https://docsify.js.org/#/) website to display docs in a nice way through github pages.
`docsify` loads `.md` files dynamically from `docs/` and displays them in a nice looking way with a sidebar. 
[Example of a docsify website](https://docs.snipaste.com/)

To deploy it on our repo, we need to enable github pages as in [this tutorial](https://docsify.js.org/#/deploy).

Changes:
- `docs/index.html` - Entry point for the documentation website.
- `docs/nojekyll` - needed for docsify to work correctly on github as per [this](https://docsify.js.org/#/quickstart?id=writing-content).
- `package.json` - new dep: `docsify-cli` + command `yarn run docs:serve`.

Use `yarn run docs:serve` to serve the website and view it locally.

### CODE REVIEW

**Comment by dev-matan-tsuberi** [comment]
Closing. Docs are now on the wiki.

**Comment by leviadam** [comment]
Can we change `gen` to something more meaningful such as generate?

**Comment by dev-matan-tsuberi** [comment]
@leviadam irrelevant to this PR, but sure https://github.com/daostack/daostack/pull/255

File: package.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
     "solium": "solium --dir ./contracts/",
     "truffle": "truffle",
     "build": "rimraf build && truffle compile",
+    "docs:serve": "node ./node_modules/docsify-cli/bin/docsify serve ./docs",
```
Comment (leviadam) [inline package.json]: I think this can change to "docsify serve ./docs"

**Comment by dev-matan-tsuberi** [comment]
a couple more fine tunings and i'll squash & merge

**Comment by dev-matan-tsuberi** [comment]
Merging, Can you turn on github pages via `docs` at `settings`?

**Pull request merged**